CONTRIBUTING.md

```diff
@@ -1,18 +1,19 @@
 # Development environment
 
 To develop tetra, you will need:
- * to install Bundler and Ruby development headers. On SUSE distros, use `sudo zypper install ruby2.5-rubygem-bundler ruby2.5-devel`;
- * to get development dependencies: `bundle install`;
+
+* to install Bundler and Ruby development headers. On SUSE distros, use `sudo zypper install ruby2.5-rubygem-bundler ruby-devel`;
+* to get development dependencies: `bundle install`;
 
 To install a development version of tetra use:
 
-```
+```bash
 sudo rake install
 ```
 
 To run tests, simply run `rake`. Please note that tests are divided into fine (more similar to unit tests) and coarse (more similar to integration tests).
 
-# Release
+## Release
```

Comment:
To me this title is on the same level as the previous, so I would recommend to either use title-1 or title-2 for both.

Reply:
Makes sense to me. The markdown linter is not that happy when using 2 primary headlines. So either way it will complain. Burt maybe the first heading could be `Contributing` and then the other 2 are just subheadings.